src/app/mytools/sidebar-accordion/sidebar-accordion.component.html

```diff
@@ -1,8 +1,8 @@
 <mat-accordion>
-  <mat-expansion-panel *ngFor="let orgObj of groupEntriesObject" [expanded]="orgObj | expandPanel: (toolId$ | async)">
+  <mat-expansion-panel *ngFor="let orgObj of groupEntriesObject" [expanded]="orgObj.expanded">
     <mat-expansion-panel-header>
       <mat-panel-title class="org-accordion-header">
-        {{ orgObj?.namespace }}
+        {{ orgObj.registry + '/' + orgObj?.namespace }}
```

Comment:
Missing ?

Reply:
It's impossible for it to be undefined. The 2nd one is redundant, but oh well.